inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/ClickHouseLoadNode.java

```diff
@@ -73,12 +85,21 @@ public ClickHouseLoadNode(@JsonProperty("id") String id,
             @Nonnull @JsonProperty("tableName") String tableName,
             @Nonnull @JsonProperty("url") String url,
             @Nonnull @JsonProperty("userName") String userName,
-            @Nonnull @JsonProperty("passWord") String password) {
+            @Nonnull @JsonProperty("passWord") String password,
+            @JsonProperty("engine") String engine,
+            @JsonProperty("partitionBy") String partitionBy,

```

Comment:
why add `partitionBy`、`orderBy`、`engine` params?  Where are they used？

Reply:
These are important attributes of clickhouse. Even if you don't use them directly now, you can add them first.

Reply:
`partitionBy` can be replaced by `partitionFields` such as `HiveLoadNode`.

Reply:
> why add `partitionBy`、`orderBy`、`engine` params? Where are they used？

ok, allright , although it is useless temporally.

Reply:
These properties have been removed